main.py

```diff
@@ -16,5 +16,24 @@ def index():
     return render_template(template_filename, **context_dict)
 
 
+@app.route("/search", methods=["GET"])
+def search():
+    app.logger.info("search(): GET /search")
+
+    keyword = "サーカスTC"
```

Comment:
```suggestion
    keyword = request.args.get("query", "")
```
と、requestのクエリを受け取ってください（PR#26 でやっている内容です）

Reply:
承知しました。
#26 の内容をそのまま使います。